docs/api-docs/channels/channel-app-best-practices.md

```diff
@@ -1,65 +1,78 @@
 
+<!-- https://developer.bigcommerce.com/api-docs/channels/guide/channel-app-best-practices -->
+
 <div class="otp" id="no-index">
 
-### On this Page
+## On this page
 
-- [Modular Features](#modular-features)
-- [Syncing](#syncing)
-- [Performance](#performance)
-- [Logging](#logging)
-- [Related resources](#related-resources)
+  - [Modular features](#modular-features)
+  - [Syncing](#syncing)
+  - [Performance](#performance)
+  - [Logging](#logging)
+  - [Related resources](#related-resources)
 
 </div>
 
 To enable the best user experience, below are a few items to keep in mind when developing a BigCommerce Channel App.
 
-## Modular Features
+## Modular features
 
 As the specific needs of every merchant is different, the features of your app should be as modular as possible.
 
-### For Merchants
+### For merchants
+
 Consider the audience and various use cases for your application:
-* **Merchant A**: only interested in importing products and inventory.
-* **Merchant B**: interested in a bi-directional integration of products and sales that keeps data between the channel and BigCommerce in sync.
+
+- **Merchant A**: only interested in importing products and inventory.
+- **Merchant B**: interested in a bi-directional integration of products and sales that keeps data between the channel and BigCommerce in sync.
 
 Merchants should have the option to opt-in or out of the various features of your application.
 
-### For Support
-If an issue occurs, a merchant should have the option to disable a specific feature that may not be behaving as expected.  The idea here is to allow the merchant to continue using the features that are functional while temporarily resolving an issue - instead of having to disable the application completely.
+### For support
+
+If an issue occurs, a merchant should have the option to disable a specific feature that may not be behaving as expected. The idea here is to allow the merchant to continue using the features that are functional while temporarily resolving an issue - instead of having to disable the application completely.
 
 ## Syncing
+
 For the various features of your application, it is typically a best practice to offer both a manual 'one time sync' option and automated sync options (based on various sync intervals).
 
 This will provide the merchant with more control over what is synced and when.
 
-**Example Automated Sync Intervals:**
-* **Do not update automatically**:  The merchant should have an option to opt-out of automatic syncs.
-* **Every X Minutes**:  This would trigger the associated event every x minutes.
-* **Real Time**:  This would utilize webhooks to provide ‘real time’ updates.
+Example Automated Sync Intervals:
+
+- **Do not update automatically**: The merchant should have an option to opt-out of automatic syncs.
+- **Every X Minutes**: This would trigger the associated event every x minutes.
+- **Real Time**: This would utilize webhooks to provide ‘real time’ updates.

```

Comment:
```suggestion
- **Real Time**: This would utilize webhooks to provide ‘real-time’ updates.

```

Reply:
@uwikaiddi Should we add hyphens to both, then? Also, does it need quotation marks?

`**Real-Time**: This would utilize webhooks to provide real-time updates.`

Reply:
@markcmurphy 
No, we only hyphenate things acting as descriptors/adjectives. 
What kind of update? A **real-time** update....
And yes, remove the quotations.